src/content/perf/ui-performance.md

```diff
@@ -4,50 +4,47 @@ subtitle: Where to look when your Flutter app drops frames in the UI.
 description: Diagnosing UI performance issues in Flutter.
 ---
 
-{% render docs/performance.md %}
-
-:::secondary What you'll learn
-* Flutter aims to provide 60 frames per second (fps) performance,
-  or 120 fps performance on devices capable of 120Hz updates.
-* For 60fps, frames need to render approximately every 16ms.
-* Jank occurs when the UI doesn't render smoothly. For example,
-  every so often, a frame takes 10 times longer to render,
-  so it gets dropped, and the animation visibly jerks.
-:::
+## Overview
 
-It's been said that "a _fast_ app is great,
-but a _smooth_ app is even better."
-If your app isn't rendering smoothly,
-how do you fix it? Where do you begin?
-This guide shows you where to start,
-steps to take, and tools that can help.
+App performance encompasses various aspects, from raw speed and I/O throughput
```

Comment:
Oooh, I like this. I wrote that OG paragraph years ago and when I re-reviewed it yesterday, I cringed. :)

Reply:
@sfshaza2, I actually loved how this read! There were so many additions to this guide over the years and I ended up going with a more generic writing style to capture them.